cygnus-ngsi/doc/installation_and_administration_guide/management_interface.md

```diff
@@ -272,6 +273,72 @@ Responses:
 
 ```
 400 Bad Request
-{"error":"
+{"error":"}
+```
+
+[Top](#top)
+
+##<a name="section10"></a>`POST /v1/subscriptions`
+
+Creates a new subscription to Orion, passed as a Json in the payload (Two JSON are required, the first must contain
```

Comment:
Creates a new subscription to Orion. The Json passed in the payload contains the Json subscription itself and Orion's endpoint details.